### ISSUE TITLE
PQ value is something wrong.

### ISSUE THREAD

**Issue by songhwanjun** [facebookresearch/detr#474]
I used the DETR-R50 pre-trained on COCO panoptic. 
When evaluating the panoptic performance, SQ (~74) is similar to the reported performance, but QP (~25) is much smaller than the reported one. The below is my evaluation command, please check it.

python -m torch.distributed.launch --nproc_per_node=8 --use_env main.py --masks --eval --coco_path /path/to/coco  --coco_panoptic_path /path/to/coco_panoptic  --dataset_file coco_panoptic --resume detr-r50-panoptic-00ce5173.pth
